data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_spells.lst

```diff
@@ -109,9 +109,9 @@ Slowing Mud					CLASSES:Druid,Shaman=4										SCHOOL:Conjuration	SUBSCHOOL:Cre
-Spellcrash					CLASSES:Cleric,Sorcerer,Wizard,Witch=6							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.193	DESC:This spell functions as lesser spellcrash, except that it affects 5th-level or lower prepared spell or spell slot.|PRERULE:1,DisplayFullSpell
-Spellcrash (Greater)			CLASSES:Cleric,Sorcerer,Wizard,Witch=8							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.194	DESC:This spell functions as lesser spellcrash, except that it affects a 7th-level or lower prepared spell or spell slot.|PRERULE:1,DisplayFullSpell
-Spellcrash (Lesser)			CLASSES:Cleric,Sorcerer,Wizard,Witch=4							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.194	DESC:You create a discordant blast of energy that disrupts the target's available magic. If the target prepares spells, it must choose one of its prepared 3rd-level spells, which is immediately lost; if the target has no 3rd-level spells prepared, it must lose a 2ndlevel spell it has prepared (progressing down to 1st-level spell if it has no 2nd-level spells prepared). If the target is a spontaneous spellcaster, it loses one of its available 3rd-level spell slots; if the target has no available 3rd-level spell slots, it must lose a 2nd-level spell slot (progressing down to 1st-level spell if it has no 2ndlevel spell slots available). If the target has more than one spellcasting class, choose one at random to be affected. This spell has no effect on spell-like abilities. Any spell or spell slot lost because of this spell is treated as if the caster had failed a concentration check while trying to cast it-the spell or spell slot is wasted and has no effect, but is recovered normally the next time the character prepares spells or regains spell slots.|PRERULE:1,DisplayFullSpell
+Spellcrash					CLASSES:Cleric,Sorcerer,Wizard,Witch=6							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:(CASTERLEVEL) rounds								SAVEINFO:Will negates (see text)													SPELLRES:yes					SOURCEPAGE:p.193	DESC:This spell functions as lesser spellcrash, except that it affects 5th-level or lower prepared spells or spell slots.|PRERULE:1,DisplayFullSpell
```

Comment:
I checked and do not see any spaces in the spellcrash spells except in the
text portions of the data (i.e DESC, DESCRIPTION, etc.)

On Mon, Oct 19, 2015 at 9:19 PM, Andrew Maitland <notifications@github.com>
wrote:

> In
> data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_spells.lst
> <https://github.com/PCGen/pcgen/pull/668#discussion_r42448827>:
>
> > @@ -109,9 +109,9 @@ Slowing Mud					CLASSES:Druid,Shaman=4										SCHOOL:Conjuration	SUBSCHOOL:Cre
> >  Sonic Form					CLASSES:Alchemist,Bard,Sorcerer,Wizard=6							SCHOOL:Transmutation	SUBSCHOOL:Polymorph	DESCRIPTOR:Sonic						COMPS:V, S, M (a pinch of dust gathered from a gravestone or a sacred shrine)										CASTTIME:1 standard action	RANGE:Personal					TARGETAREA:you																							DURATION:(CASTERLEVEL) rounds																														SOURCEPAGE:p.193	DESC:Upon casting this spell, you keep your relative shape, but you and your equipment are transformed into semisolid sound. While in this sonic form, you take no penalties for squeezing, and you can move through spaces without penalty as if you were a creature three size categories smaller. You can make a melee touch attack once per round that deals 1d6 points of sonic damage + 1 point per caster level. You are considered incorporeal (although you cannot move through solid objects). Any nonmagical attacks you make deal half damage. Magical attacks are unaffected, and you can use your magic items and other equipme
>  nt as normal. You cannot enter an area of silence. If you are in the area of a silence spell, you take 1d6 points of sonic damage per caster level of the silence spell in the round it is cast and in each round when you end your turn within the spell's area. You can attempt a Fortitude saving throw each round to take half damage. If this spell's duration ends when you are in a square that a creature of your size cannot normally occupy, you take 3d6 points of damage and are shunted to the nearest open space that you can normally occupy.|PRERULE:1,DisplayFullSpell
> >  Sonic Scream				CLASSES:Bard,Bloodrager,Magus,Sorcerer,Wizard=2						SCHOOL:Evocation						DESCRIPTOR:Sonic						COMPS:V, S																					CASTTIME:1 standard action	RANGE:15 Ft.					TARGETAREA:cone-shaped burst																					DURATION:(CASTERLEVEL) rounds or until discharged [see text]	SAVEINFO:Reflex half (see text)											SPELLRES:yes					SOURCEPAGE:p.193	DESC:Up to three times during this spell's duration, you can emit a cone of powerful sound as a standard action. The first cone deals 4d4 points of sonic damage to every creature in the area. The second deals 2d4 points of sonic damage to every creature in the area. The third deals 1d4 points of sonic damage to every creature in the area. A successful Reflex save against a cone halves that cone's damage. After the third cone of sound, the spell ends. Until you use all three cones, you cannot use your voice for any magic-related purpose (bardic performance, verbal spellcasting, and so on), and cannot spe
>  ak at any volume lower than a shout.|PRERULE:1,DisplayFullSpell
> >  Speak with Haunt				CLASSES:Cleric,Witch=4|Shaman=3								SCHOOL:Necromancy						DESCRIPTOR:Language-Dependent				COMPS:V, S, DF																				CASTTIME:10 minutes		RANGE:10 Ft. [see text]				TARGETAREA:one haunt																						DURATION:(CASTERLEVEL) minutes						SAVEINFO:Will negates (see text)											SPELLRES:no						SOURCEPAGE:p.193	DESC:You stir a haunt (Pathfinder RPG GameMastery Guide 242) to a limited sense of awareness and consciousness, allowing it to answer questions. The spell's range must reach any square within 10 feet of the haunt's area. You must be aware of the haunt prior to casting the spell, and casting the spell does not trigger the haunt. You can ask one question per 2 caster levels. The haunt's knowledge is limited to what its original creature knew during life, including the languages it spoke. A haunt often remembers the circumstance that led to its existence (though this recollection might be from the original victim's pers
>  pective and therefore not objective), what triggers it, and how it can be laid to rest (destroyed). Answers are brief, cryptic, or repetitive, especially if the haunt is angry and vindictive. If the haunt's alignment is more than one step away from yours, the haunt can attempt a Will save to resist the spell. A haunt's Will save modifier is equal to 3 + the haunt's CR. If the save is successful, the haunt can refuse to answer your questions or attempt to deceive you (using Bluff). A haunt's Bluff modifier equals its CR (minimum +0) or might be determined by the GM based upon the original victim. The haunt can speak only about what it knew in life and the circumstances by which it became a haunt. It cannot answer any questions that pertain to events that occurred after it was created. A neutral or good haunt might cooperate with similarly aligned creatures in order to end its suffering. If a haunt has been subject to this spell within the past week, a new casting of t
>  his spell on it fails. You can cast this on a haunt that has been deceased for any amount of time. Unlike a corpse affected by speak with dead, a haunt wants to express itself, if only to share its pain or to cause mischief.|PRERULE:1,DisplayFullSpell
> > -Spellcrash					CLASSES:Cleric,Sorcerer,Wizard,Witch=6							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.193	DESC:This spell functions as lesser spellcrash, except that it affects 5th-level or lower prepared spell or spell slot.|PRERULE:1,DisplayFullSpell
> > -Spellcrash (Greater)			CLASSES:Cleric,Sorcerer,Wizard,Witch=8							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.194	DESC:This spell functions as lesser spellcrash, except that it affects a 7th-level or lower prepared spell or spell slot.|PRERULE:1,DisplayFullSpell
> > -Spellcrash (Lesser)			CLASSES:Cleric,Sorcerer,Wizard,Witch=4							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.194	DESC:You create a discordant blast of energy that disrupts the target's available magic. If the target prepares spells, it must choose one of its prepared 3rd-level spells, which is immediately lost; if the target has no 3rd-level spells prepared, it must lose a 2ndlevel spell it has prepared (progressing down to 1st-level spell if it has no 2nd-level spells prepared). If the target is a spontaneous spellcaster, it loses one of its available 3rd-level spell slots; if the target has no available 3rd-level spell slots, it must lose a 2nd-level spell slot (progressing down to 1st-level spell if it has no 2ndlevel spell slots available). If the target has more than on
>  e spellcasting class, choose one at random to be affected. This spell has no effect on spell-like abilities. Any spell or spell slot lost because of this spell is treated as if the caster had failed a concentration check while trying to cast it-the spell or spell slot is wasted and has no effect, but is recovered normally the next time the character prepares spells or regains spell slots.|PRERULE:1,DisplayFullSpell
> > +Spellcrash					CLASSES:Cleric,Sorcerer,Wizard,Witch=6							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:(CASTERLEVEL) rounds								SAVEINFO:Will negates (see text)													SPELLRES:yes					SOURCEPAGE:p.193	DESC:This spell functions as lesser spellcrash, except that it affects 5th-level or lower prepared spells or spell slots.|PRERULE:1,DisplayFullSpell
>
> There is an unexplained "space" that appears to be a tab or two for the
> spellcrash spells, can you verify this is not a space.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PCGen/pcgen/pull/668/files#r42448827>.
>